akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/ReplicatedDataSerializer.scala

```diff
@@ -37,23 +37,26 @@ private object ReplicatedDataSerializer {
   abstract class KeyComparator[A <: GeneratedMessage] extends Comparator[A] {
 
     /**
-     * Get the key from the entry. The key may be a String, Integer, Long, or Any
+     * Get the key from the entry. The key may be a String, Int, Long, or OtherMessage
      * @param entry The protobuf entry used with Map types
      * @return The Key
      */
     def getKey(entry: A): Any
     final def compare(x: A, y: A): Int = compareKeys(getKey(x), getKey(y))
     private final def compareKeys(t1: Any, t2: Any): Int = (t1, t2) match {
       case (k1: String, k2: String)             => k1.compareTo(k2)
-      case (k1: String, k2)                     => -1
-      case (k1, k2: String)                     => 1
+      case (_: String, _)                       => -1
+      case (_, _: String)                       => 1
       case (k1: Int, k2: Int)                   => k1.compareTo(k2)
-      case (k1: Int, k2)                        => -1
-      case (k1, k2: Int)                        => 1
+      case (_: Int, _)                          => -1
+      case (_, _: Int)                          => 1
       case (k1: Long, k2: Long)                 => k1.compareTo(k2)
-      case (k1: Long, k2)                       => -1
-      case (k1, k2: Long)                       => 1
+      case (_: Long, _)                         => -1
+      case (_, _: Long)                         => 1
```

Comment:
I kind of feel that it is a shame to loose these because the names helped a little to read it, even if they weren't actually used